components/org.wso2.carbon.identity.api.server.certificate.validation.management/org.wso2.carbon.identity.api.server.certificate.validation.management.v1/src/main/java/org/wso2/carbon/identity/api/server/certificate/validation/management/v1/impl/CertificateValidationApiServiceImpl.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.certificate.validation.management.v1.impl;
+
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.CertificateValidationApiService;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.core.ServerCertificateValidationManagementService;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.factories.ServerCertificateValidationManagementServiceFactory;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.CACertificateAddRequest;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.CACertificateUpdateRequest;
+import org.wso2.carbon.identity.api.server.certificate.validation.management.v1.model.Validator;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * Certificate Validation API service implementation.
+ */
+public class CertificateValidationApiServiceImpl implements CertificateValidationApiService {
+
+    private final ServerCertificateValidationManagementService certificateValidationManagementService;
+
+    public CertificateValidationApiServiceImpl() {
+
+        try {
+            certificateValidationManagementService = ServerCertificateValidationManagementServiceFactory
+                    .getServerCertificateValidationManagementService();
+        } catch (IllegalStateException e) {
+            throw new RuntimeException("Error occurred while initiating the certificate validation management service.",
+                    e);
+        }
+    }
+
+    @Override
+    public Response addCACertificate(CACertificateAddRequest caCertificateAddRequest) {
+
+        return Response.ok().entity(certificateValidationManagementService
```

Comment:
Is it correct to return `ok()` always?

Reply:
Most of the application api also used ok by passing the status inside 
https://github.com/wso2/identity-api-server/blob/master/components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/impl/ApplicationsApiServiceImpl.java#L108-L137